CameraExtrinsicMat:
   rows: 4
   cols: 4
   dt: d
   data: [0.940484191615406 ,  -0.338088415197291 ,  0.0344341230598409 ,  0.0205870940136826,
          0.110149329537504 ,  0.207409032103355 ,   -0.972033239454489 ,  -0.105625714536456,
          0.321491229311087 ,  0.917974790999913 ,   0.232305128149836 ,   0.0537288192257312,
                0.,  0.,  0.,  1.]
CameraMat:
   rows: 3
   cols: 3
   dt: d
   data: [ 538.39128993937,   0.,   308.049009633327,
           0.,  539.777910345317,  248.065244763797, 
           0., 0.,   1. ]
DistCoeff:
   rows: 1
   cols: 5
   dt: d
   data: [0.1066, -0.1792, 0.,   0.,   0.]
ImageSize: [ 640,  480 ]
ReprojectionError: 0
DistModel: plumb_bob